karma.conf.js

```diff
@@ -0,0 +1,48 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = function (config) {
+
+  // To specify browsers to test against use BROWSER var and comma separated list:
+  // BROWSER=Firefox,Chrome,Safari,IE npm run test:browser
+  var browsers = process.env.BROWSER ? process.env.BROWSER.split(',') : [ 'Firefox' ];
```

Comment:
No, you can't do `npm run test:browser --browsers Chrome,Firefox`.

However, it seems that since npm 2 you **can** do `npm run test:browser -- --browsers Chrome,Firefox` which I had no idea had been added until 5 mins ago. Great!